### DISCUSSION TITLE
Is it possible to remove the duck from the Finder?

### DISCUSSION THREAD

**Discussion by dsmith4-godaddy** [iterate-ch/cyberduck#15936] [Q&A] [answered]
After I installed Mountain Duck, my Finder now has a duck (not the one in the menu bar, I'd expect that one to be there):

<img width="998" alt="Screenshot 2024-05-14 at 2 19 53 PM" src="https://github.com/iterate-ch/cyberduck/assets/109608083/124eb524-9da8-4639-97c8-fdaf103867bc">

This duck is present even when Mountain Duck is not running, which is a bit confusing. And, honestly, I'd prefer to keep it only in the menu bar. Is there a setting to hide or remove this Finder duck?

**Comment by dkocher** [comment, marked as answer]
Choose _View → Customize Toolbar…_ or `command`-drag the icon out of the toolbar.

**Reply by dsmith4-godaddy** [reply]
Appreciate the quick response! (I wish it weren't added by default, but I may be in the minority in that opinion.) Worked perfectly, thank you!
